ui/src/main/resources/local.properties

```diff
@@ -0,0 +1,7 @@
+url=https://localhost:8080
+
+user=default
+password=1q2w3e
+
+browser=firefox
+timeout=30
```

Comment:
need to discuss the property management,and loading

Reply:
commom app prop file

Reply:
updated